test-browser/tests/terminal.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
     browser
     .waitForElementVisible('*[data-id="terminalCli"]')
     .click('*[data-id="terminalCli"]')
-    .keys('remix.')
+    .sendKeys('*[data-id="terminalCliInput"]', 'remix.')
```

Comment:
`sendKeys` works for chrome too?
so maybe we don't need https://github.com/ethereum/remix-ide/pull/2626/files#diff-bce217bd304a22784cfcdae7bd97b11aR10

Reply:
Yes, i'll take it out and make them uniform.